docs/source/user_manual/basic_elements/plot_renderers.rst

```diff
@@ -258,20 +258,20 @@ All :class:`~chaco.abstract_plot_renderer.AbstractPlotRenderer` subclasses are
 expected to provide three methods for mapping to and from screen space and
 data space:
 
-    :method:`map_screen`
+    :attr:`map_screen`
```

Comment:
`map_screen`, `map_data` and `map_index` are methods. It would be better to be tags as methods. The problem is probably that the `:method:` role links to a method and does not define one. (e.g.
```
.. :py:method: map_screen(<signature>) 

```
should do the right thing